docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

```diff
@@ -1,11 +1,45 @@
 # Converting a PyTorch Model {#openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch}
 
-The PyTorch framework is supported through export to the ONNX format. In order to optimize and deploy a model that was trained with it:
+The PyTorch framework is supported through export to the ONNX format. Model Optimizer Python API allows the conversion of PyTorch models using the `convert_model()` method, which internally converts a model to ONNX.
+
+## Converting a PyTorch model from memory with Python API (Experimental Functionality)
+
+To convert a PyTorch model using `convert_model()`, provide the `input_shape` or `example_input`.
+
+```sh
+import torchvision
+import torch
+from openvino.tools.mo import convert_model
+
+model = torchvision.models.resnet50(pretrained=True)
+ov_model = convert_model(model, example_input=torch.zeros(1, 3, 100, 100))
+```
+
+'example_input' accepts the following formats:
+
+* `openvino.runtime.Tensor`
+* `torch.Tensor`
+* `np.ndarray`
+* `list` or `tuple` with tensors (`openvino.runtime.Tensor` / `torch.Tensor` / `np.ndarray`)
+* `dictionary` where key is the input name, value is the tensor (`openvino.runtime.Tensor` / `torch.Tensor` / `np.ndarray`)
+
+ONNX opset version can be set using an optional `onnx_opset_version` parameter.
```

Comment:
Please note that this works in case of `use_new_frontend=False` if we choose using PT FE by default. Or PT FE by default approach is problematic, then note that this option always implies ONNX path and it may not be backward compatible in case when we switch to a different default approach.

Reply:
Currently `use_new_frontend` does not switch PyTorch conversion path yet.
When https://github.com/openvinotoolkit/openvino/pull/15197 is implemented this part of documentation will need updating.